host/inc/miscellaneous.h

```diff
@@ -100,57 +104,44 @@ int strincmp(const char*, const char*, int);
 
 class LocalRNG
 {
-	uint32_t Q[4096], i, c; // CMWC4096 variables
+        unsigned int state;
+        
+        void Init(unsigned int seed)
+        {
+            state = seed;
+        }
+        
+
+        public:
+        LocalRNG(unsigned int seed)
```

Comment:
That's the only additional function we should add to LocalRNG - with the argument being a uint32_t